tools/istio-iptables/pkg/builder/iptables_builder_impl.go

```diff
@@ -180,7 +181,16 @@ func (rb *IptablesRuleBuilder) BuildV6() [][]string {
 
 func (rb *IptablesRuleBuilder) constructIptablesRestoreContents(tableRulesMap map[string][]string) string {
 	var b strings.Builder
-	for table, rules := range tableRulesMap {
+	keys := make([]string, 0, len(tableRulesMap))
```

Comment:
We already do something similar in `cni/pkg/plugin/plugin_dryrun_test.go`, I think you can just do

`tables := slices.Sort(maps.Keys(generatedRules))`

Reply:
That's definitely cleaner :)
Changed